src/cljs/main/broadfcui/page/workspace/details.cljs

```diff
@@ -19,7 +20,7 @@
   {:render
    (fn [{:keys [props]}]
      (let [{:keys [workspace]} props]
-       (when (and workspace (get-in workspace [:workspace :realm]))
+       (when (and workspace (= (get-in workspace [:workspace :authorizationDomain :usersGroupName]) (config/dbgap-authorization-domain)))
```

Comment:
**Before:** Showed the protected TCGA data banner for all protected workspaces
**After:** Shows the protected TCGA data banner for only workspaces protected by the dbGapAuthorizedUsers auth domain